packages/frontend/src/components/message/MessageListAndComposer.tsx

```diff
@@ -95,26 +105,96 @@ export default function MessageListAndComposer({ accountId, chat }: Props) {
     regularMessageInputRef
   )
 
-  const onDrop = async (e: React.DragEvent<any>) => {
-    if (chat === null) {
-      log.warn('dropped something, but no chat is selected')
-      return
+  useEffect(() => {
+    const unset = runtime.setDragListener(async e => {
+      if (e.payload.type != 'drop') {
+        return
+      }
+      if (chat === null) {
+        log.warn('dropped something, but no chat is selected')
+        return
+      }
+
+      // sanitize files
+      const paths = e.payload.paths
+      const forbiddenPathRegEx = /DeltaChat\/.+?\.sqlite-blobs\//gi
```

Comment:
Can we avoid duplication? And BTW I think `sqlite-blobs` is no longer used.

https://github.com/deltachat/deltachat-desktop/blob/9e7a6a1e815ca66a6cfe0df5787107cda428973d/packages/target-electron/runtime-electron/runtime.ts#L139